src/lib/djerba/core/main.py

```diff
@@ -324,6 +324,46 @@ def setup(self, assay, ini_path, compact):
                 'supplement.body',
                 'wgts.snv_indel'
             ]
+        elif assay == 'WGS':
+            component_list = [
+                'core',
+                'input_params_helper',
+                'provenance_helper',
+                'gene_information_merger',
+                'treatment_options_merger',
+                'case_overview',
+                'cnv',
+                'sample',
+                'summary',
+                'supplement.header',
+                'supplement.body',
+                'wgts.snv_indel'
+             ]
+	elif assay == 'TAR':
+            component_list = [
+                'core',
+                'tar_input_params_helper',
+                'provenance_helper',
+                'gene_information_merger',
+                'treatment_options_merger',
+                'case_overview',
+                'tar.sample',
+                'tar.swgs', 
+                'summary',
+                'supplement.header',
+                'supplement.body',
+                'tar.snv_indel'
+            ]
+        elif assay == 'pWGS':
```

Comment:
Should we make this ALL CAPS, to be consistent with allowed assay names in the input params helper? https://github.com/oicr-gsi/djerba/blob/fbf730e66fab835182724a89bdbdb6c15be91d1f/src/lib/djerba/helpers/input_params_helper/helper.py#L39